LibreNMS/OS/Ewc.php

```diff
@@ -127,4 +138,167 @@ public function discoverWirelessClients()
         }
         return $sensors;
     }
+
+    /**
+     * Discover wireless bit errors.  This is in total bits. Type is errors.
+     * Returns an array of LibreNMS\Device\Sensor objects that have been discovered
+     *
+     * @return array Sensors
+     */
+    public function discoverWirelessErrors()
+    {
+        $oids = snmpwalk_cache_oid($this->getDevice(), 'apPerfRadioPktRetx', array(), 'HIPATH-WIRELESS-HWC-MIB');
+        $ap_interfaces = $this->getCacheByIndex('apName', 'HIPATH-WIRELESS-HWC-MIB');
+
+        $sensors = array();
+        foreach ($oids as $index => $entry) {
+            $name = $ap_interfaces[explode('.', $index)[0]];
+            $sensors[] = new WirelessSensor(
+                'errors',
+                $this->getDeviceId(),
+                '.1.3.6.1.4.1.4329.15.3.5.2.5.1.18.' . $index,
+                'ewc',
+                $index . 'Retx',
+                "Retransmits ($name radio " . explode('.', $index)[1]. ")"
+            );
+        }
+        return $sensors;
+    }
+
+    /**
+     * Discover wireless frequency.  This is in MHz. Type is frequency.
+     * Returns an array of LibreNMS\Device\Sensor objects that have been discovered
+     *
+     * @return array Sensors
+     */
+    public function discoverWirelessFrequency()
+    {
+        $oids = snmpwalk_cache_oid($this->getDevice(), 'apRadioStatusChannel', array(), 'HIPATH-WIRELESS-HWC-MIB');
+        $ap_interfaces = $this->getCacheByIndex('ifName', 'IEEE8021-BRIDGE-MIB');
```

Comment:
IEEE8021-BRIDGE-MIB does not contain ifName.
Probably try to avoid fetching again unless you don't have the data